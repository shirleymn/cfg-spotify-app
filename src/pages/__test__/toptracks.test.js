
import TopTracks from '../toptracks'

import renderer from "react-test-renderer";



it("matches snapshot", () =>{
    const tree = renderer.create(<TopTracks></TopTracks>).toJSON();
    expect(tree).toMatchSnapshot();

});