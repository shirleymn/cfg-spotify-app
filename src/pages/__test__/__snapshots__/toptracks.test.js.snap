// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <h1>
    Your Top Tracks
  </h1>
  <button
    className="text-button"
    onClick={[Function]}
  >
    Last 4 Weeks
  </button>
  <button
    className="text-button"
    onClick={[Function]}
  >
    Last 6 Months
  </button>
  <button
    className="text-button"
    onClick={[Function]}
  >
    All Time
  </button>
  <br />
   
  <br />
</div>
`;
